flow/src/test/java/com/sequenceiq/flow/core/Flow2HandlerTest.java

```diff
@@ -150,6 +152,12 @@
     @Mock
     private SpanContext spanContext;
 
+    @Mock
```

Comment:
are these fields not called at all in the tests ? I guess some verification should be useful

Reply:
wrote a test